src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -0,0 +1,827 @@
+                    :rules="Rules.LastNameRules"
+                    />
+                </div>
+              </v-form>
+            </v-col>
+          </v-row>
+          <v-row no-gutters class="mt-0">
+            <v-col id="resolution-signature-card-left-col" cols="2" class="pt-6" >
+              <v-card-title class="resolution-signature-vcard-title pl-2 pr-0">
+                Date Signed
+              </v-card-title>
+            </v-col>
+            <v-col id="resolution-signature-card-right-col" cols="10" class="pt-4 pl-6 pr-5">
+              <date-picker
+                ref="signatureDatePickerRef"
+                title="Date Signed"
+                nudge-right="40"
+                :initialValue="signatureDateText"
+                :minDate="signatureDateMinStr"
+                :maxDate="signatureDateMaxStr"
+                :inputRules="signatureDateRules"
+                @emitDateSync="onSigningDateSync"
+              />
+            </v-col>
+          </v-row>
+        </v-card>
+      </div>
+    </section>
+
+    <section id="confirm-resolution-section" class="mt-10">
+      <header id="resolution-confirm-header">
+        <h2>{{getCreateResolutionResource.confirmSection.header}}</h2>
+      </header>
+      <div :class="{ 'invalid-section': getShowErrors && !hasResolutionConfirmed }">
+        <v-card flat id="confirm-resolution-card" class="mt-4 pt-10 pb-8 pl-6 pr-9">
+          <v-form ref="confirmResolutionChkFormRef">
+            <v-checkbox
+              ref="confirmResolutionChkRef"
+              class="chk-resolution ml-1"
+              id="chk-confirm-resolution"
+              v-model="resolutionConfirmed"
+              :rules="confirmCompletionResolution"
+              @change="onResolutionConfirmedChange($event)"
+            >
+              <div slot="label" v-html="confirmLabel"></div>
+            </v-checkbox>
+            <ul>
+              <li class="mt-1"
+                v-for="(item, index) in getCreateResolutionResource.confirmSection.items"
+                :key="index">
+                <v-row v-if="item.type === ItemTypes.TEXT" no-gutters>
+                  <v-col cols="auto" class="pt-1 mr-2"><v-icon>mdi-circle-small</v-icon></v-col>
+                  <v-col cols="11" class="pt-1 pb-1" no-gutters>
+                    {{item.value}}
+                  </v-col>
+                </v-row>
+                <v-row v-if="item.type === ItemTypes.PARTIAL_ITEMS" no-gutters>
+                  <v-col cols="auto" class="pt-1 mr-2"><v-icon>mdi-circle-small</v-icon></v-col>
+                  <v-col cols="11" class="pt-1 pb-1" no-gutters>
+                    <span v-for="(partialItem, index) in item.value" :key="index">
+                      <span v-if="partialItem.type === ItemTypes.TEXT" v-html="partialItem.value"></span>
+                      <span v-if="partialItem.type === ItemTypes.PLACEHOLDER
+                                && partialItem.value === PLACEHOLDER_LEGAL_NAME_INLINE">
```

Comment:
Maybe preference, not sure, but in the past the `&&` operator would be on the initial line before the line break. 
Super small thing

Reply:
What Cam says is enforced by our linter in script code.

Reply:
yup, i can fix it.

Reply:
should it pick it up when I run `npm run lint`?  doesn't pick it up currently.  no big deal anyways.  was just curious.

Reply:
Sadly (or thankfully), our linter doesn't much look at the template code. I think it identifies long lines (sometimes) and that's about it. 🤷‍♂️

Reply:
probably thankfully...haha

Reply:
updated